### ISSUE TITLE
Summary needs to show uploaded file name.

### ISSUE THREAD

**Issue by watarutmnh** [verbb/formie#737]
**Description**
Summary not showing uploaded file name.

**Steps to reproduce**
1. Create 2 pages Form.
2. Add a file upload field in Page 1 and a summary field to Page 2.
3. Upload a file and check Summary output at frontend.

### Form settings
- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

**Additional info**
- Plugin version: dev-craft-3 as 1.5.7
- Craft version: 3.7.30.1
- Multi-site: Yes

**Comment by engram-design** [comment]
File upload fields should have their value shown in a summary field as a link. For example:

![image](https://user-images.githubusercontent.com/1221575/151474756-d104d126-b9b3-411d-bf8d-ab332f0631b0.png)

```
<a href="/uploads/test/beth-smith-pltSxmU1Vg0-unsplash_2022-01-28-020633_aryh.jpg">beth-smith-pltSxmU1Vg0-unsplash_2022-01-28-020633_aryh.jpg</a>
```

Don't forget you can use the [`modifyValueForSummary`](https://verbb.io/craft-plugins/formie/docs/developers/events#the-modifyValueForSummary-event) event to change Formie's defaults

**Comment by watarutmnh** [comment]
@engram-design
So that mean you can't reproduce the issue?
I haven't changed Formie's defaults so I should have that link.

![Screen Shot 2022-01-28 at 17 56 56](https://user-images.githubusercontent.com/6081378/151517112-f39ce8f2-dce5-46be-bde3-c27ada4ad281.png)

**Comment by engram-design** [comment]
Hmm, I see what you mean. Does the asset volume you're uploading images to have public URLs enabled?

I've just done a quick recording https://d.pr/v/mACDPO

**Comment by watarutmnh** [comment]
@engram-design
Oh Sorry I got it, of course there is no link since it is not public URLs enabled!
I disabled public URLs  for a security reason.

I think it is better to have only a file name without a link when it has no public URLs if possible.

**Comment by engram-design** [comment]
Great point! Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by watarutmnh** [comment]
Thanks for the fix!

**Comment by engram-design** [comment]
Fixed in [1.5.8](https://github.com/verbb/formie/releases/tag/1.5.8)
